js/tabs.js

```diff
@@ -465,26 +469,13 @@ Tabs.prototype.saveEntry_ = function(tab, entry, opt_callback) {
   this.save(tab, opt_callback);
 };
 
-Tabs.prototype.onDocChanged_ = function(e, session) {
+Tabs.prototype.onDocChanged_ = function(_, { type }) {
+  // Assume that updates are coming from the current tab, if they arn't
+  // we have no real way to recover so failing here is acceptable.
   var tab = this.currentTab_;
-  if (this.currentTab_.getSession() !== session) {
-    console.warn('Something wrong. Current session should be',
```

Comment:
why are we removing this warning message? even if failure is acceptable a warning might be good

Reply:
To be honest the warning messages was unreliable, it would fire during normal function as well as when it was warranted. I didn't want to add the extra logic to the editors to maintain a copy of their own session to pass with every docChange event. If you feel it's worth having happy to put it back in, i just didn't feel it was adding much.

Reply:
After some thought i'm gonna put this back in, seems like it's doing more good then bad, thanks for pointing it out :)

Reply:
Ack.